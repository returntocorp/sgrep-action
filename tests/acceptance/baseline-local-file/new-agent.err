| versions          - semgrep x.y.z on Python x.y.z
| environment       - running in environment git, triggering event is 'unknown'
| manage            - not logged in
=== setting up agent configuration
| using semgrep rules from .semgrep.yml
| using default path ignore rules of common test and dependency directories
| reporting findings introduced by these commits:
|   * aaaaaaa add rules
| looking at 2 changed paths
| found 2 files in the paths to be scanned
=== looking for current issues in 2 files
| 1 current issue found
| No ignored issues found
=== .semgrep.yml file not found in baseline, skipping scanning for baseline
=== not looking at pre-exiting issues since after filtering out local files that don't exist in baseline, no configs left to run
=== exiting with failing status
