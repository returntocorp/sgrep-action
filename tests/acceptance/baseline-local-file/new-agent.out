python.lang.correctness.useless-eqeq.useless-eqeq
     > bar.py:1
     ╷
    1│   3 == 3
     ╵
     = This expression is always True: `3 == 3` or `3 != 3`. If testing for
       floating point NaN, use `math.isnan(3)`, or `cmath.isnan(3)` if the
       number is complex.

